assets/components/payPalContributionButton/payPalContributionButton.scss

```diff
@@ -42,9 +42,17 @@
   &:hover {
     border-color: #000;
     color: #000;
-    
+
     .svg-arrow-right-straight {
       fill: #000;
     }
   }
+}
+
+.component-paypal-contribution-button--bundle-landing {
+  width: 100%;
+
+  &.contrib-error {
+    top: -16px;
+  }
```

Comment:
Am I understanding this correctly that you only want to apply these styles when the button appears on the bundles landing page?

We've set up our CSS so that these shared components should ideally know nothing about the context in which they are used. Then context-specific styles are applied through the page-level stylesheets.

For example, in this case, this component is shared across both the bundles landing and contributions landing pages. Any styles that are generic, part of the component regardless of where it's used, sit in `payPalContributionButton.scss`. Then, any styles that want to modify it on the specific pages where it is used are placed in the stylesheets for those pages. The page stylesheets always sit below the component stylesheets in the load order, so page-level styles always override component-level styles.

To summarise my ramblings, I think these style rules should sit within `bundlesLanding.scss`, e.g. [here](https://github.com/guardian/support-frontend/blob/master/assets/pages/bundles-landing/bundlesLanding.scss#L217) 😊:

```scss
.component-paypal-contribution-button {
  width: 100%;

  &.contrib-error {
    top: -16px;
  }
}
```

Reply:
Thanks Jamie - I've moved that around and it's much clearer.